### DISCUSSION TITLE
Am I exposing my IP address by using Beaker?

### DISCUSSION THREAD

**Discussion by interstar** [beakerbrowser/beaker#1818] [General]
A privacy / security question. 

When I publish a document, can that be traced back to my home IP address?

**Comment by KyGost** [comment]
I do not know enough about Beaker and its underlying functions to comment on this but I believe the answer is yes.
![image](https://user-images.githubusercontent.com/12588894/94359267-6f52a800-00e9-11eb-9f7f-184e9bb4c341.png)
(see beaker://settings/?view=network)

**Comment by pfrazee** [comment]
The answer is yes, it's required for people to fetch the data from your device. We've talked about proxies (distributed and non-distributed) to hide IPs but don't yet have a solution we're ready to run with.
